pkg/get/get_test.go

```diff
@@ -3118,3 +3118,50 @@ func Test_DownloadHostcl(t *testing.T) {
 		}
 	}
 }
+
+func Test_DownloadKubecm(t *testing.T) {
+	tools := MakeTools()
+	name := "kubecm"
+	version := "0.16.2"
```

Comment:
If you remove the `tool.Version` as I suggested you probably need to change this to: 
```suggestion
	version := "v0.16.2"
```

Reply:
> 

modified in new commit [378e192](https://github.com/karthick-kk/arkade/commit/378e19209bbd68a3dd4246ed90a67d4b4b82af40)